volttrontesting/platform/auth_tests/test_auth_control.py

```diff
@@ -724,3 +558,169 @@ def _remove_known_host(platform, host):
         p.communicate()
         assert p.returncode == 0
 
+
+@pytest.fixture()
+def mock_auth_service():
+    AuthService.__bases__ = (AgentMock.imitate(Agent, Agent()), )
+    yield AuthService(
+        auth_file=MagicMock(), protected_topics_file=MagicMock(), setup_mode=MagicMock(), aip=MagicMock())
+
+
+@pytest.fixture()
+def test_auth():
```

Comment:
This should not have a test prefix or pytest may get confused by it.  Change it to auth_config_fixture or something like this.